helm/charts/determined/templates/master-config.yaml

```diff
@@ -55,6 +55,8 @@ data:
       {{- end }}
       {{- end }}
 
+    {{$cpuImage := (split "/" "determinedai/environments:py-3.7-pytorch-1.7-tf-1.15-cpu-606fd02")._1}}
```

Comment:
does that need `{{-` and `-}}` to avoid another empty line? lgtm otherwise.

Reply:
I tested this several times and adding the ``{{-`` or ``-}}`` to that would cause there to be no newline at all. It ends up looking the most correct as is.